### DISCUSSION TITLE
Design for Planet

### DISCUSSION THREAD

**Discussion by YiHanCA** [Planetable/Planet#89] [Show and tell]
gm ☀️ My name is Yi, and I am the designer working on Planet. Today I would love to share some designs I did for the project. Feedbacks are welcome! ❤️

![planet_web_logo](https://user-images.githubusercontent.com/39843151/196257557-ad9d4cdf-e72d-499b-9b2f-3332be776aac.png)

![planet_app_icon](https://user-images.githubusercontent.com/39843151/196259977-6912278a-2030-49fd-beb0-3a1d7bb106cf.png)

![planet_wallpaper_1](https://user-images.githubusercontent.com/39843151/196265845-c3d471be-a3de-4c90-ac81-97cad02c96f6.png)

![planet_wallpaper_2](https://user-images.githubusercontent.com/39843151/196265860-059a9c08-2cf8-485d-bf96-985f87ce6eb3.png)

**Comment by pilvijo** [comment]
The dark blue one looks fantastic! Did you use Ethereum-design colors? Looks alike and still is unique!

**Reply by YiHanCA** [reply]
Thank you, Pilvijo. I didn't choose Ethereum colors. I used purple because it feels magical :)

**Reply by pilvijo** [reply]
So y are a wizard of colors as well, capable of feeling them. I think it has something to do with the evolution of humankind. So much hidden potential. But now back to daily routines and thanks for the design.
